### PR TITLE
Added Database insertion and retrieval for Registration Page.

### PR DESCRIPTION
**Pull request by mikco-hipe**
OVERVIEW

- added modifications to RegisterController to insert data from the Registration page to database
- added UsersInformation model to define the relationship between user table and users_information table
- added initial VerifyEmailNotifications for email verification (WIP)
- fixed the remaining naming inconsistencies as it causes errors
NOTES
- working data insertion and retrieval for Registration

### CODE REVIEW

**Review by hipe-francis — COMMENTED** [review]
I don't think the created_at and updated_at fileds should be validated that way, Please refer to the Laravel docs for the your reference.

File: laravel-app/app/Models/UsersInformation.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    protected $fillable = [
+        'first_name',
+        'last_name',
+    ];
+
+    public function user()
+    {
+        return $this->belongsTo(User::class);
+    }
+}
```
Comment (hipe01mark) [inline laravel-app/app/Models/UsersInformation.php:21]: Add new line!

File: laravel-app/routes/web.php [hunk 2]
```diff
@@ -16,6 +16,4 @@
 
 Auth::routes();
 
-Route::get('/', [HomeController::class, 'index'])->name('home');
-Route::get('/home', [HomeController::class, 'index'])->name('home');
-
+Route::get('/', [HomeController::class, 'index'])->name('home');
```
Comment (hipe01mark) [inline laravel-app/routes/web.php:19]: Add new line!

File: laravel-app/app/Http/Controllers/Auth/RegisterController.php [hunk 3]
```diff
@@ -50,9 +51,14 @@ public function __construct()
     protected function validator(array $data)
     {
         return Validator::make($data, [
```
Comment (hipe01mark) [inline laravel-app/app/Http/Controllers/Auth/RegisterController.php:53]: Do not expose your validation requests in your controller. You should use **form request** in laravel

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

File: laravel-app/app/Http/Requests/CreateUserRequest.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateUserRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
```
Comment (bpocrafael) [inline laravel-app/app/Http/Requests/CreateUserRequest.php:12]: do you think we need this authorize function?

File: laravel-app/app/Http/Requests/CreateUserRequest.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
```
Comment (bpocrafael) [inline laravel-app/app/Http/Requests/CreateUserRequest.php:20]: nil, remov comments

File: laravel-app/app/Notifications/VerifyEmailNotification.php [hunk 6]
```diff
@@ -0,0 +1,54 @@
+<?php
```
Comment (bpocrafael) [inline laravel-app/app/Notifications/VerifyEmailNotification.php:1]: too many unnecessary code

File: laravel-app/app/Http/Requests/CreateUserRequest.php (hunk 4 above)
Comment (mikco-hipe) [reply laravel-app/app/Http/Requests/CreateUserRequest.php:12]: Based sir on the reference that I saw, it is needed to perform authorization to make requests of new users.

File: laravel-app/app/Http/Requests/CreateUserRequest.php (hunk 5 above)
Comment (mikco-hipe) [reply laravel-app/app/Http/Requests/CreateUserRequest.php:20]: Copy that sir

File: laravel-app/app/Notifications/VerifyEmailNotification.php (hunk 6 above)
Comment (mikco-hipe) [reply laravel-app/app/Notifications/VerifyEmailNotification.php:1]: Yes sir, removing it now.

File: laravel-app/app/Http/Requests/CreateUserRequest.php (hunk 4 above)
Comment (bpocrafael) [reply laravel-app/app/Http/Requests/CreateUserRequest.php:12]: so right now do you think this function authorize has been used??

Comment (mikco-hipe) [reply laravel-app/app/Http/Requests/CreateUserRequest.php:12]: Upon testing and reading the code sir, turns out function authorize was not used. I removed it and it does not affect the code.

**Review by bpocrafael — APPROVED** [review]

**Pull request merged**